lib/src/controls/navigation/navigation_view/view.dart

```diff
@@ -34,11 +38,14 @@ class NavigationView extends StatefulWidget {
     Key? key,
     this.appBar,
     this.pane,
+    this.content,
     this.clipBehavior = Clip.antiAlias,
     this.contentShape,
     this.onOpenSearch,
     this.transitionBuilder,
-  }) : super(key: key);
+  })  : assert((pane != null && content == null) ||
+            (pane == null && content != null)),
```

Comment:
Add an error message:

```suggestion
  })  : assert(
    (pane != null && content == null) || (pane == null && content != null),
    'Either pane or content must be provided',
    ),
```